fe1-web/features/meeting/screens/CreateMeeting.tsx

```diff
@@ -16,7 +16,7 @@ import ConfirmModal from 'components/ConfirmModal';
 import DatePicker, { onChangeStartTime, onChangeEndTime } from 'components/DatePicker';
 import { Timestamp } from 'model/objects';
 import { FOUR_SECONDS } from 'res/const';
-import { onConfirmPress } from 'parts/lao/organizer/eventCreation/CreateEvent';
+import { onConfirmPress } from 'features/events/screens/CreateEvent';
```

Comment:
we'll probably want to move this to a (future) `common` package, independent of the `features/events`, or accept the code duplication 🤔

Reply:
Yes I think moving it in `common` is nice. Maybe we could have something like `common/events` for things like this that are common among all events